h2o-r/h2o-package/R/parse.R

```diff
@@ -140,6 +147,8 @@ h2o.parseSetup <- function(data, pattern="", destination_frame = "", header = NA
   # begin the setup
   # setup the parse parameters here
   parseSetup.params <- list()
+  
+  if(!is.null(custom_non_data_line_markers)) {parseSetup.params$custom_non_data_line_markers = custom_non_data_line_markers}
```

Comment:
unnecessary {}, I would either remove the {} or put the statement on a new line

Reply:
Style guidelines do not forbid it (https://style.tidyverse.org/syntax.html#indenting - section 2.4.1) . However, the lines was **too long**, so I chose the s econd option.